### DISCUSSION TITLE
2019 mobility data

### DISCUSSION THREAD

**Discussion by Maryam-taf** [ActiveConclusion/COVID19_mobility#57] [Q&A] [answered]
Hi Folks, is there any 2019 available from Apple, Waze, etc?

**Comment by ActiveConclusion** [comment, marked as answer]
Hi @Maryam-taf!
Unfortunately, Apple and Waze data are no longer available.

> https://covid19.apple.com/mobility 
As of April 14, 2022, Apple is no longer providing COVID-19 mobility trends reports.
<img width="562" alt="image" src="https://user-images.githubusercontent.com/63144882/184019880-d92d17cb-3f3c-47ab-91b6-3a8ac3f9d023.png">

> https://datastudio.google.com/u/0/reporting/fe8a3c7d-9303-4e70-8acb-4e042714fa76/page/bhuOB
Waze | COVID-19 Impact Dashboard has been retired and will no longer be updated as of July 2022
<img width="499" alt="image" src="https://user-images.githubusercontent.com/63144882/184020149-2bc5bc9a-5d76-4c33-a316-6fa70dc178ee.png">

Apple data starts from 01.2020. Waze data is available from 03.2020.

**Reply by Maryam-taf** [reply]
Thanks for the response!
